### PR TITLE
Parallax and theoretical relations

### PR DESCRIPTION
**Pull request by Alex-J-Brown**
I've added a few new bits:
   - blackbody.py and limbdark.py now uses filter specific tables and limbdark uses Claret's new WD values, I haven't included quadratic or linear laws at the moment but will do that soon if you like.
    - filters.py is a generic and basic module but helps out for filter transmissions and effective wavelengths
    - massradius.py does what it sounds like, for WDs can choose between helium and CO core and for M dwarfs, can use an empirical relation or baraffe tracks.
    - rochedistortion.py just makes an interpolator for going from volume averaged radius to radius along centres for a given mass ratio (all scaled radii).
    - I've altered lcurve.py a bit so that a scalefactor can be given for use with absolute fluxes.

### CODE REVIEW

File: pylcurve/lcurve.py [hunk 1]
```diff
@@ -124,11 +150,13 @@ def write(self, fname=None):
...
 
-    def run(self, data):
+
+    def run(self, data, scale_factor=None):
         """
-        Run lroche, return output and name of model
+        Run lroche, return output and name of model.
         """
         data = str(data)
+        scale_factor = str(scale_factor)
```
Comment (StuartLittlefair) [inline pylcurve/lcurve.py]: How does this work for scale_factor=None?

Comment (StuartLittlefair) [reply pylcurve/lcurve.py]: I don't think the string conversion is needed

File: pylcurve/utils.py [hunk 2]
```diff
@@ -1,13 +1,74 @@
...
-from astropy import units as u
+import astropy.units as u
+from scipy.integrate import quad
 from astropy.constants import G
+from astropy.modeling.blackbody import blackbody_lambda, blackbody_nu
+from .blackbody import bb_interpolator
+from .limbdark import ld_interpolator
+from .massradius import mr_interpolator
+from .rochedistortion import roche_interpolator
+from dust_extinction.parameter_averages import F20
```
Comment (StuartLittlefair) [inline pylcurve/utils.py:10]: dust_extinction? Is this an external package?

File: scripts/lcmcmc_fc.py [hunk 3]
```diff
@@ -0,0 +1,343 @@
+import emcee
```
Comment (StuartLittlefair) [inline scripts/lcmcmc_fc.py:1]: add a docstring at the top of this file briefly explaining how the model works

**Review by StuartLittlefair — CHANGES_REQUESTED** [review]
Looks good Alex - I've requested a few minor changes and then I'll merge it back in

File: pylcurve/lcurve.py (hunk 1 above)
Comment (Alex-J-Brown) [reply pylcurve/lcurve.py]: Fixed (I think), the string conversion was accidentally left behind after i put format in there

File: pylcurve/utils.py (hunk 2 above)
Comment (Alex-J-Brown) [reply pylcurve/utils.py:10]: Yeah it's an external package

File: scripts/lcmcmc_fc.py (hunk 3 above)
Comment (Alex-J-Brown) [reply scripts/lcmcmc_fc.py:1]: Done, give us a shout if it's not along the lines you were after

**Pull request merged**